Client/Frontend/Browser/TabManager.swift

```diff
@@ -122,6 +120,7 @@ class TabManager : NSObject {
         }
 
         assert(tab === selectedTab, "Expected tab is selected")
+        selectedTab?.resurrect()
```

Comment:
`resurrect` is a bit of a misnomer -- you're "resurrecting" tabs that are brand new. Maybe `initializeWebView` would be a more appropriate name?